lib/daemon_controller.rb

```diff
@@ -571,10 +565,7 @@ def determine_lock_file(given_lock_file, identifier, pid_file)
   def run_command(command)
     if should_capture_output_while_running_command?
       # Create tempfile for storing the command's output.
-      tempfile = Tempfile.new("daemon-output")
-      tempfile.chmod(0o666)
-      tempfile_path = tempfile.path
-      tempfile.close
+      tempfile_path = Tempfile.create("daemon-output").tap(&:close).path
```

Comment:
I don't see the point in using `Tempfile.create`. `Tempfile#close` also doesn't guarantee returning the tempfile object. I think we should stick with the old code.